src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -179,27 +180,26 @@ def _call_rp_configure_cicd(
     json_request = req.json()
     return json_request
 
-def list_releases(name, resource_group_name):
+def list_releases(target_name, target_resource_group):
     """
     Lists all the release definitions that are deployed to a given Azure container service.
 
-    :param name: Name of the target Azure container service instance.
-    :type name: String
-    :param resource_group_name: Name of Azure container service's resource group.
-    :type resource_group_name: String
+    :param target_name: Name of the target Azure container service instance.
+    :type target_name: String
+    :param target_resource_group: Name of Azure container service's resource group.
+    :type target_resource_group: String
     """
     profile = Profile()
     cred, subscription_id, _ = profile.get_login_credentials()
 
     o_auth_token = cred.signed_session().headers['Authorization']
+    container_service_resource_id = CONTAINER_SERVICE_RESOURCE_URL.format(subscription_id=subscription_id, resource_group_name=target_resource_group, container_service_name=target_name)
     data = {
-        'token': o_auth_token
+        'acsResourceId': container_service_resource_id
     }
 
-    list_releases_url = RP_URL.format(
-        subscription_id=subscription_id,
-        resource_group_name=resource_group_name,
-        container_service_name=name) + '/configureCI/1/listReleases?api-version=' + API_VERSION
+    list_releases_url = SERVICE_URL.format(
+        subscription_id=subscription_id) + '/listReleases?api-version=' + API_VERSION
 
     headers = {'Content-type': 'application/json', 'Authorization': o_auth_token}
     req = requests.post(list_releases_url, data=json.dumps(data), headers=headers, timeout=600)
```

Comment:
It's unusual to have calls to requests in a CLI command. I haven't seen it anywhere else in the CLI. Is there not an SDK call that does this?